WordPress/Classes/ViewRelated/Reader/WPFriendFinderViewController.m

```diff
@@ -1,275 +0,0 @@
-#import <AddressBook/AddressBook.h>
-#import <Accounts/Accounts.h>
-#import <Social/Social.h>
```

Comment:
Looks like this was the only file using those three frameworks, maybe we can stop linking them.